### PR TITLE
Update source.rst

### PR DESCRIPTION
**Pull request by martinshelton**
Minor updates to the copy

## Status

Ready for review / Work in progress

## Description of Changes

* Description: 

* Fixes Issue#

## Testing

*

## Release 

* 

## Checklist (Optional)

- [ ] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [ ] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/source.rst
Comment (rocodes) [inline docs/source.rst:95]: nit: minor typo here ("landing age" instead of "landing page").  I'll leave it to others whether or not we stick to the caps + italicized glossary-style "_Landing Page_" or not.

File: docs/source.rst
Comment (rocodes) [inline docs/source.rst:123]: while we're in here:
``sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion/report-an-error``

**Comment by rocodes** [comment]
Thanks @martinshelton! I think these changes improve clarity from a source's point of view. I will wait for one more reviewer's feedback on this, but generally LGTM. 

A small nitpick: we try to stick to ~80-character line limits (if you look at the source diff view, you'll see the difference). If you have time to adjust line lengths, and do a quick rebase, that would be great, but if not we can take care of it.

File: docs/source.rst
Comment (eloquence) [inline docs/source.rst:23]: Per our docs guidelines, would be good to format all new text to 80 characters width for consistency. I can add a commit to do that @martinshelton if you want, once all other comments have been resolved.

**Review by eloquence — CHANGES_REQUESTED** [review]
(See initial comments by @rocodes and myself, let's resolve these before merging.)

File: docs/source.rst
Comment (eloquence) [reply docs/source.rst:95]: See the related recommendation to link defined terms to the glossary: https://github.com/freedomofpress/securedrop-docs/issues/54

In this case, the glossary provides very little additional information, so I'm fine with not italicizing the term, and not linking it to the glossary.

**Comment by eloquence** [comment]
@martinshelton wants to try to get this over the line, but we may want to do a git workshop session together before then. Removing from our project board for now, but will pick this up again soon.

**Comment by rocodes** [comment]
Since this branch had gotten a bit stale and @martinshelton was working on his `main`, he and I worked together to grab his changes and stick them on another branch so that he could rebase + squash like a beast.  

https://github.com/freedomofpress/securedrop-docs/pull/148 replaces this PR.

**Pull request closed without merging**
